src/views/PopupView/LoginPopup/LoginPopup.tsx

```diff
@@ -1,18 +1,18 @@
 import React, {useEffect, useState} from 'react';
 import './LoginPopup.scss';
-import {Input} from '@material-ui/core';
-import {connect} from 'react-redux';
-import {AppState} from '../../../store';
-import {updateAuthData} from '../../../store/auth/actionCreators';
-import {updateActivePopupType} from '../../../store/general/actionCreators';
-import {GenericYesNoPopup} from '../GenericYesNoPopup/GenericYesNoPopup';
-import {AuthData} from '../../../store/auth/types';
-import {ClipLoader} from 'react-spinners';
-import {CSSHelper} from '../../../logic/helpers/CSSHelper';
-import {APIService} from '../../../services/API';
-import {PopupWindowType} from '../../../data/enums/PopupWindowType';
+import { Input } from '@material-ui/core';
```

Comment:
[minor]
Please keep the previous code format. It would be nice to focus on which part of the codes changed.

Reply:
@choipd, I was change code to previous in this view.